packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -156,9 +166,9 @@ describe('executePlugins', () => {
       {
         'output.json': JSON.stringify([
           {
-            slug: 'node-version',
-            score: 0.3,
-            value: 16,
+            slug: nodePluginSlug,
+            score: 1,
+            value: 2,
```

Comment:
Why were the values changed?

Now the value is Node version 2.0.0 (not realistic) yet the score is 100%. That does not make sense.

Reply:
Resolved but not addressed.